Dockerfile

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG GO_VERSION="1.23"
+ARG GO_VERSION="1.24"
```

Comment:
https://github.com/golang/go/issues/73144 has not been released. Are we sure this is not affected?

Reply:
hum yeah indeed, moving to draft for now

Reply:
It seems the Go issue has been closed recently, maybe it's safe to merge this PR? Syft scanner has fixed a bug in this release that would make our lives much easier if it's rolled out.

Reply:
Yup, go 1.24.3 has the fix :tada: